packages/beacon-node/src/metrics/metrics/lodestar.ts

```diff
@@ -728,6 +728,38 @@ export function createLodestarMetrics(
         buckets: [0.05, 0.1, 0.2, 0.5, 1, 1.5, 2, 4],
       }),
     },
+    gossipDataColumn: {
+      recvToValidation: register.histogram({
+        name: "lodestar_gossip_data_column_received_to_gossip_validate",
```

Comment:
unit is missing

Reply:
the metric name might also suggest that it tracks time between receiving until before validation, not including validation time